features/org.wso2.identity.apps.authentication.portal.server.feature/resources/web.xml.j2

```diff
@@ -0,0 +1,536 @@
+  ~ software distributed under the License is distributed on an
+  ~ "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+  ~ KIND, either express or implied.  See the License for the
+  ~ specific language governing permissions and limitations
+  ~ under the License.
+  -->
+
+<web-app xmlns="http://java.sun.com/xml/ns/javaee"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://java.sun.com/xml/ns/javaee http://java.sun.com/xml/ns/javaee/web-app_3_0.xsd"
+         version="3.0" metadata-complete="true">
+    <absolute-ordering />
+
+    <!--  Custom Page configurations -->
+    <!-- *************** Application specific configurations ********************** -->
+    <!--context-param>
+        <param-name>ServiceProviderApp-/samlsso_login.do</param-name>
+        <param-value>http://localhost:8080/customauthenticationendpoint/login.do</param-value>
+    </context-param-->
+    <!--context-param>
+        <param-name>ServiceProviderApp-/oauth2_authz.do</param-name>
+        <param-value>http://localhost:8080/authenticationendpoint/oauth2_authz.do</param-value>
+    </context-param-->
+    <!--context-param>
+        <param-name>ServiceProviderApp-/oauth2_error.do</param-name>
+        <param-value>http://localhost:8080/authenticationendpoint/oauth2_error.do</param-value>
+    </context-param-->
+    <!--context-param>
+        <param-name>ServiceProviderApp-/oauth2_consent.do</param-name>
+        <param-value>http://localhost:8080/authenticationendpoint/oauth2_consent.do</param-value>
+    </context-param-->
+    <!-- **************** End of Application specific configurations ************************* -->
+
+    <!-- *************** Global configurations ********************** -->
+    <!--context-param>
+        <param-name>/retry.do</param-name>
+        <param-value>http://localhost:8080/customauthenticationendpoint/retry.do?type=retry.do</param-value>
+    </context-param-->
+    <!-- *************** End of Global configurations ********************** -->
+
+    <!-- *************** Account Recovery Endpoint Context URL Configuration ********************** -->
+    <!--context-param>
+       <param-name>IdentityManagementEndpointContextURL</param-name>
+       <param-value>https://localhost:9443/accountrecoveryendpoint</param-value>
+   </context-param-->
+    <context-param>
+       <param-name>AccountRecoveryRESTEndpointURL</param-name>
+       <param-value>/t/tenant-domain/api/identity/user/v1.0/</param-value>
+   </context-param>
+    <!--context-param>
+        <param-name>EnableRecoveryEndpoint</param-name>
+        <param-value>true</param-value>
+    </context-param-->
+    <!--context-param>
+        <param-name>EnableSelfSignUpEndpoint</param-name>
+        <param-value>true</param-value>
+    </context-param-->
+
+    {% if authenticationendpoint.registration_url is defined %}
+    <context-param>
+        <param-name>AccountRegisterEndpointURL</param-name>
```

Comment:
Is this endpoint used in on-prem product as well?

Reply:
Yes. That url is used in `Create Account` buttons of the login form. Recently it is modified to configure it from the web.xml (in #1496)